src/main/java/homeworks/fivesection/enumpractice/Professions.java

```diff
@@ -3,55 +3,63 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static org.apache.commons.lang3.StringUtils.INDEX_NOT_FOUND;
+
 public enum Professions {
-    PLUMBER("Hard Worker", 1), ACTOR(2), DEV("Crying man"), QA(4), SPY(5);
+    PLUMBER("Hard Worker", 1),
+    ACTOR(2),
+    DEV("Crying man"),
+    QA(4),
+    SPY(5),
+    TOASTER;
 
-    private String profName;
-    private Integer value;
+    private Integer number;
 
-    public String getProfName() {
-        return profName;
-    }
-
-    public Integer getValue() {
-        return value;
+    Professions() {
     }
 
-    public String getProfNameAsString() {
-        return String.valueOf(value);
+    Professions(String profName) {
+        this.profName = profName;
     }
-    public String getProfIntegerAsString() {
 
-        return String.valueOf(profName);
+    Professions(Integer number) {
+        this.number = number;
     }
 
-    // empty constructors
+    private String profName;
 
-    Professions() {
+    Professions(String profName, Integer number) {
+        this.profName = profName;
+        this.number = number;
     }
 
-    //constructors for String
-    Professions(String profName) {
-        this.profName = profName;
+    public String getProfName() {
+        return profName;
     }
 
-    //constructors for Integer
-    Professions(Integer value) {
-        this.value = value;
+    public static Professions convertIntegerToProf(Integer value) {
+        for (Professions professions : Professions.values()) {
+            if (professions.getNumber().equals(value)) {
+                return professions;
+            }
+        }
+        return null;
     }
 
-    //constructors for String and Integer
-    Professions(String profName, Integer value) {
-        this.profName = profName;
-        this.value = value;
+    public static Integer convertProfToInteger(Professions inputProf) {
```

Comment:
No needs in this one, just inputProf.getNumber,and  if you would like to have INDEX_NOT_FOUND than it could be 
inputProf.getNumber==null? INDEX_NOT_FOUND :inputProf.getNumber